webapp/src/views/CrudDomainList.vue

```diff
@@ -76,7 +76,9 @@ export default {
               store.commit('working', false);
             }
           }
-          this.extraComponentBind = {'name': d.name, 'ds': d.keys.map(key => key.ds)};
+          let ds = d.keys.map(key => key.ds);
+          ds = ds.concat.apply([], ds)
```

Comment:
what does this do?

Reply:
it flattens the list. `d.keys` is the `keys` field specified [here](https://desec.readthedocs.io/en/latest/dns/domains.html#domain-field-reference), and the resulting `ds` is a list of strings from `d.keys[*]['ds']`, where the list index `*` is "integrated over"; it does not matter how may entries the `keys` field has.